extensions/cornerstone-dicom-pmap/src/getSopClassHandlerModule.ts

```diff
@@ -119,6 +119,22 @@ function _getDisplaySetsFromSeries(
   return [displaySet];
 }
 
+const getRangeFromPixelData = (pixelData: Float32Array) => {
```

Comment:
Does this need to include pixel padding range, and then exclude that from the range calculations?

Reply:
is there any pixel padding in pmap?

Reply:
Seems like pmap does not have pixel padding